bin/main/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.application.name=skills-service
+eureka.client.serviceUrl.defaultZone=http://ec2-52-22-238-103.compute-1.amazonaws.com:8761/eureka
+eureka.instance.preferIpAddress=true
+server.port=8082
```

Comment:
As a good practice try to live a whitespace at the end of the files

Reply:
I agree, I did not take into consideration that to generate a withespace, I should give double line break and not just one.